src/core/MOM_CoriolisAdv.F90

```diff
@@ -227,6 +245,19 @@ subroutine CorAdCalc(u, v, h, uh, vh, CAu, CAv, OBC, AD, G, GV, US, CS, pbv, Wav
   real :: QUHeff,QVHeff ! More temporary variables [H L2 T-2 ~> m3 s-2 or kg s-2].
   integer :: i, j, k, n, is, ie, js, je, Isq, Ieq, Jsq, Jeq, nz
   logical :: Stokes_VF
+  real :: u_v, v_u, q_v, q_u ! u_v is the u velocity at v point, v_u is the v velocity at u point
```

Comment:
All real variables need to be fully described in their comments, including their units.  The units are missing for `u_v` and `v_u` (which should be `[L T-1 ~> m s-1]`, and for `q_v` and `q_u` the missing units should be `[H-1 T-1 ~> m-1 s-1 or m2 kg-1 s-1]`.  The units for `h_v' and `h_u' should be `[H ~> m or kg m-2]`.  The units are give for `Ih_sum`, but they are incorrect when in non-Boussinesq mode, so the units should be `H-1 ~> m-1 or m2 kg-1`.  Units are also needed for `Ih_third`, `Ih_fifth` and `Ih_seventh`.